domains/auth/Auth.API/API/Models/Oidc/SignaturGruppen/UserInfoToken.cs

```diff
@@ -0,0 +1,60 @@
+using System.Text.Json.Serialization;
+
+namespace API.Models;
+#nullable disable
+public record UserInfoToken
+{
+    [JsonPropertyName("iss")]
+    public string Iss { get; init; }
```

Comment:
There are some similarities with `IdTokenInfo` and this record. Would it make sense to make an abstract class?